api/Api/EnvironmentManager/Design.cs

```diff
@@ -0,0 +1,606 @@
+﻿using SharpGLTF.IO;
+using SharpGLTF.Memory;
+using SharpGLTF.Schema2;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using UnityEngine;
+
+namespace SynthesisAPI.EnvironmentManager
+{
+    public class Design
```

Comment:
The classes within this one should be moved to separate files. They can be created in a `Design` folder within a `Design` namespace.